presto-main/src/test/java/com/facebook/presto/operator/scalar/TestVarbinaryFunctions.java

```diff
@@ -511,4 +511,13 @@ private static SqlVarbinary varbinary(int... bytesAsInts)
         }
         return new SqlVarbinary(bytes);
     }
+
+    @Test
+    public void testReverse()
```

Comment:
Also add a couple of tests that check: 

reverse(reverse(x)) = x

Reply:
Added